### PR TITLE
fix(curriculum): Add solutions to Applied Visual Design challenges

### PR DESCRIPTION
**Pull request by huyenltnguyen**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Related to #35751 

In this PR, I:
- Added solutions to many Applied Visual Design challenges
- Replaced invalid solutions with working ones
- Formatted some challenge seed (mostly indentation)

### CODE REVIEW

**Comment by RandellDawson** [comment]
How many invalid solutions did you find?

**Comment by huyenltnguyen** [comment]
**Challenges with invalid solution (15):**
- add-a-box-shadow-to-a-card-like-element.english.md
- adjust-the-tone-of-a-color.english.md
- animate-elements-at-variable-rates.english.md
- center-an-element-horizontally-using-the-margin-property.english.md
- create-a-gradual-css-linear-gradient.english.md
- create-a-graphic-using-css.english.md
- create-movement-using-css-animation.english.md
- create-texture-by-adding-a-subtle-pattern-as-a-background-image.english.md
- create-visual-direction-by-fading-an-element-from-left-to-right.english.md
- modify-fill-mode-of-an-animation.english.md
- use-a-css-linear-gradient-to-create-a-striped-element.english.md
- use-the-css-transform-property-skewx-to-skew-an-element-along-the-x-axis.english.md
- use-the-css-transform-property-skewy-to-skew-an-element-along-the-y-axis.english.md
- use-the-css-transform-scale-property-to-change-the-size-of-an-element.english.md
- use-the-css-transform-scale-property-to-scale-an-element-on-hover.english.md

**Challenges that I formatted (3):**
- adjust-the-hover-state-of-an-anchor-tag.english.md (Removed extra new lines)
- create-a-more-complex-shape-using-css-and-html.english.md (Corrected code indentation)
- use-the-s-tag-to-strikethrough-text.english.md (Corrected code indentation)

---
@RandellDawson So that is 32 + 15 + 3 = 50 files changed.

**Review by RandellDawson — APPROVED** [review]
I validated all the solutions work as intended.  Although I would not have indented the css in the `style` elements, it is perfectly valid to do so.  There were several files where the solution was fine in the style section and the only thing changed by the contributor was the indentation.   

That being said, I am fine with these changes being merged.

File: curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-a-more-complex-shape-using-css-and-html.english.md [hunk 1]
```diff
@@ -92,40 +92,41 @@ tests:
...
-  content: "";
-  background-color: pink;
-  border-radius: 50%;
-  position: absolute;
-  width: 50px;
-  height: 50px;
-  top: -25px;
-  left: 0px;
-}
+<style>
```
Comment (moT01) [inline curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-a-more-complex-shape-using-css-and-html.english.md]:
```suggestion
```

**Review by moT01 — CHANGES_REQUESTED** [review]
Found this one typo. Everything else LGTM. If you could fix that quick @huyenltnguyen we can get this merged. Thanks for contributing!

**Review by thecodingaviator — APPROVED** [review]
LGTM! Just the change @moT01 requested

File: curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-a-more-complex-shape-using-css-and-html.english.md (hunk 1 above)
Comment (RandellDawson) [reply curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-a-more-complex-shape-using-css-and-html.english.md]: @moT01, @thecodingaviator I fixed this on behalf of user.

**Comment by thecodingaviator** [comment]
@moT01 Merge if you feel it to be fine

**Review by moT01 — APPROVED** [review]

**Pull request merged**